src/main/java/co/programmers/domain/Expression.java

```diff
@@ -0,0 +1,47 @@
+package co.programmers.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import co.programmers.exception.ExceptionMessage;
+
+public class Expression {
+
+	private static final Pattern EXPRESSION_FORMAT = Pattern.compile("(\\d+\\s[\\+\\-\\*\\/]\\s)+\\d+");
+	private static final String DELIMITER = " ";
+	private String expression;
+
+	public Expression(String expression) {
+		this.expression = expression;
+		if (expression == null || expression.isEmpty()) {
+			throw new IllegalArgumentException(ExceptionMessage.EMPTY_INPUT);
+		}
+		if (!validate()) {
+			throw new ArithmeticException(ExceptionMessage.INVALID_EXPRESSION);
+		}
+	}
+
+	private boolean validate() {
+
+		Matcher matcher = EXPRESSION_FORMAT.matcher(expression);
+		return matcher.matches();
+	}
```

Comment:
```suggestion
	private boolean isNotValidate() {
		Matcher matcher = EXPRESSION_FORMAT.matcher(expression);
		return !matcher.matches();
	}
```
!를 쓰는 대신 메서드에 부정을 담으면 좀 더 명시적으로 읽힐 수 있어요!

Reply:
깃헙에 업로드 하면서 오류가 있었나봐요... 아래처럼 작성하였습니다! 혹시 `!`를 쓸일이 있으면 알려주신대로 부정문을 사용하겠습니다!

```java
	private boolean validate() {

		Matcher matcher = EXPRESSION_FORMAT.matcher(expression);
		return matcher.matches();
	}
```